model.py

```diff
@@ -794,8 +801,8 @@ def set_hyperlink(self, rel, *hrefs):
         _db = Session.object_session(self)
         hyperlink, is_modified = get_one_or_create(
             _db, Hyperlink, library=self, rel=rel,
-            create_method_kwargs=dict(href=default_href)
         )
+
         if hyperlink.href not in hrefs:
             # This is a preexisting Hyperlink whose href doesn't
             # match any of the given hrefs.
```

Comment:
I think this comment isn't quite right now. The Hyperlink may have been newly created with no href and get one here.